### PR TITLE
Ref: Ionic Docs on Capacitor and Cordova

### PR DESCRIPTION
**Pull request by lucas-zimerman**
Hide from search the mention of Ionic on Cordova.
Add an Ionic page for Capacitor.

### CODE REVIEW

File: src/platforms/javascript/guides/cordova/ionic.mdx [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+noindex: true
 redirect_from:
   - /clients/cordova/ionic/
 description: "Learn how to use Sentry with Ionic."
 ---
 
+<Alert level="warning" title="Warning">
+
+It is highly recommended to use Ionic with Capacitor, _click here_ to be redirected to the new documentation of Ionic with Sentry Capacitor.
+</Alert>
```
Comment (imatwawana) [inline src/platforms/javascript/guides/cordova/ionic.mdx:13]:
```suggestion
We strongly recommended that you use Ionic with Capacitor. Check out our [Ionic with Sentry Capacitor documentation](/platforms/javascript/guides/capacitor/ionic/.

</Alert>
```

File: src/platforms/javascript/guides/capacitor/ionic.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: Ionic
+sidebar_order: 201
+description: "Learn how to use Sentry with Ionic."
+---
+
+To use Sentry with [Ionic](https://ionicframework.com/) you have to add _sentry-capacitor_ as a dependency to you package.json.
```
Comment (imatwawana) [inline src/platforms/javascript/guides/capacitor/ionic.mdx:7]:
```suggestion
To use Sentry with [Ionic](https://ionicframework.com/), you have to add _sentry-capacitor_ as a dependency to your package.json.
```

File: src/platforms/javascript/guides/cordova/ionic.mdx (hunk 1 above)
Comment (lucas-zimerman) [reply src/platforms/javascript/guides/cordova/ionic.mdx:13]: Oh thanks for the comment, I was looking on how to properly link a page from another framework

File: src/platforms/javascript/guides/cordova/ionic.mdx [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
+noindex: true
+notoc: true
 redirect_from:
   - /clients/cordova/ionic/
 description: "Learn how to use Sentry with Ionic."
 ---
 
+<Alert level="warning" title="Warning">
+
+It is highly recommended to use Ionic with Capacitor, _click here_ to be redirected to the new documentation of Ionic with Sentry Capacitor.
```
Comment (lucas-zimerman) [inline src/platforms/javascript/guides/cordova/ionic.mdx:13]:
```suggestion
We strongly recommended that you use Ionic with Capacitor. Check out our [Ionic with Sentry Capacitor documentation](/platforms/javascript/guides/capacitor/).

```

File: src/platforms/javascript/guides/cordova/ionic.mdx [hunk 4]
```diff
@@ -1,11 +1,19 @@
 ---
 title: Ionic
 sidebar_order: 201
+noindex: true
+notoc: true
```
Comment (imatwawana) [inline src/platforms/javascript/guides/cordova/ionic.mdx:5]: For some reason this page is still showing up in the TOC in the preview. I'm going to pull this branch into my local and test it.

Comment (imatwawana) [inline src/platforms/javascript/guides/cordova/ionic.mdx:5]:
```suggestion
notoc: true
draft: false
```

Comment (lucas-zimerman) [reply src/platforms/javascript/guides/cordova/ionic.mdx:5]: Oh I didn't notice you changed that flag, I'll rollback to only alter the draft: false.

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**